templates/pages/page.html.twig

```diff
@@ -398,14 +398,11 @@
 
                                             </div>
 
-                                            {#
-                                            <div class="govuk-grid-column-one-third apollo-enclosure">
-                                                <h3 class="govuk-heading-s">Talk to us</h3>
-                                                <p>CCS customer services team is available Monday to Friday, 9am to 5pm.</p>
-
-                                                <h3 class="govuk-heading-s" style="margin-bottom: 0;">Talk to us</h3>
-                                                <p>0345 410 2222</p>
-                                            </div>#}
+                                             <div class="govuk-grid-column-one-third" ;  style=" padding:18px; background:#f8f8f8;:font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 23px" >
+								                <P class="govuk-!-font-size-14px"><b>Talk to us </b></P><br />
+								                <P >CCS customer services team is available <br />Monday to Friday, 9am to 5pm.</P><br />
+								                <P ><b>Telephone:</b><br /><a href="Tel:03454102222">0345 410 2222</a></p>
+							                </div>    
```

Comment:
This div (and the paragraphs before it) are unnecessarily indented.

Instead of this:

```
<div>
                    <p>lorem ipsum</p>
                    <p>lorem ipsum</p>
                    <p>lorem ipsum</p>
                </div>
```

Do this:

```
<div>
    <p>lorem ipsum</p>
    <p>lorem ipsum</p>
    <p>lorem ipsum</p>
</div>
```

This will make code easier to read and follow.

You should either use 4 spaces or 1 tab character to achieve this indentation (this is mostly a personal preference choice).

But the space should be equal to 4 spaces to retain consistency with other code on the site.